ezbake/config/conf.d/metrics.conf

```diff
@@ -2,5 +2,36 @@
 metrics: {
     # a server id that will be used as part of the namespace for metrics produced
     # by this server
-    server-id: "localhost"
+    server-id: localhost
+    registries: {
```

Comment:
This looks fine to me.  Could we add the same to the [dev/puppetserver.conf](https://github.com/puppetlabs/puppetserver/blob/09c359c6343af06e5b7e9b667e94f61d54b82ce5/dev/puppetserver.conf#L115-L117) file for local development?